dist/ci/docker-compose-test.sh

```diff
@@ -13,12 +13,10 @@ until curl http://api:3000/about 2>/dev/null ; do
 done
 
 cd /code
-for file in tests/*_tests.py; do
-  if ! test -f /code/.without-coverage; then
-    COVER_ARGS="--with-coverage --cover-xml --cover-package=. --cover-inclusive"
-  fi
-  echo "running tests from $file..."
-  run_as_tester nosetests $COVER_ARGS -c .noserc -s $file
+for i in tests/*_tests.py; do
```

Comment:
hmm, why did you rename my variable `file` now `i`? 

Why did you make it imposible to run without coverage now? it's a waste of time while developing locally

Reply:
ok, will revert it back.